app/services/hyrax/characterization/file_set_description.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Characterization
+    ##
+    # @api public
+    class FileSetDescription
+      include Hydra::Works::MimeTypes
+
+      ##
+      # @!attribute [rw] file_set
+      #   @return [Hyrax::FileSet]
+      attr_accessor :file_set
+
+      delegate :mime_type, to: :primary_file
```

Comment:
Can you expand on the reason for changing from the concept of original_file to primary_file?